### DISCUSSION TITLE
New blog-type-thing about metrics dashboard

### DISCUSSION THREAD

**Discussion by ahpook** [community/OSPO#78] [Dashboard]
Absolute legend that he is, @zkoppert put together a blog-type-post about using the dashboard. It's posted in the new github-ospo repo (where we've also open-sourced some policies and tools) - this will get worked into a post for the official blog before too long, but I thought it might be useful for y'all today - feedback welcome! 

https://github.com/github/github-ospo/blob/main/docs/org-metrics-dashboard.md

**Comment by ecki** [comment]
Is this available somewhere?
